Sources/Helpers/MPSImageDescriptor.swift

```diff
@@ -14,7 +14,7 @@ public extension MPSImageDescriptor {
     }
 }
 
-extension MPSImage {
+public extension MPSImage {
```

Comment:
Oh, to publicly use the new member? e.g. in the overriden version of the Start node?